package.json

```diff
@@ -66,17 +66,31 @@
   },
   "scripts": {
     "autolint": "npm run lintfix && npm run csslintfix",
+    "build:components": "cd frontend/components && npm run build",
+    "ci:all": "npm ci && cd frontend/components && npm ci",
+    "ciwin:all": "npm ci && npm run install:components",
```

Comment:
needed this as `frontend/components` is indirectly reliant on `fsevents` which is not compatible/required in windows.
`npm ci` dosen't respect this, while npm install does https://github.com/nodejs/node-gyp/issues/ 661